ledger/acctupdates.go

```diff
@@ -114,6 +114,9 @@ type accountCreatable struct {
 }
 
 type modifiedResource struct {
+	// TODO: delete
```

Comment:
This "todo" is right. There is more work here around removing this index from here and making sure getForAddress returns the index from the map's key.

Reply:
done